pocs/camera/sbigudrv.py

```diff
@@ -149,8 +164,7 @@ def assign_handle(self, serial=None):
             try:
                 index = self._handle_assigned.index(False)
             except ValueError:
-                # All handles already assigned, must be trying to intialising more cameras
-                # than are connected.
+                # All handles already assigned, must be trying to intialising more cameras than are connected.
```

Comment:
c/intialising/initialize/  (or initialise for those raised with the Queen's spelling ;-)).

Reply:
Initialise it should be :-)